pkg/apiserver/event/event.go

```diff
@@ -66,20 +66,26 @@ func Register(r *gin.RouterGroup, s *Service) {
 // @Router /api/events [get]
 // @Failure 500 {object} utils.APIError
 func (s *Service) listEvents(c *gin.Context) {
-	podName := c.Query("podName")
-	podNamespace := c.Query("podNamespace")
-	startTimeStr := c.Query("startTime")
-	experimentName := c.Query("experimentName")
-	experimentNamespace := c.Query("experimentNamespace")
-	uid := c.Query("uid")
+	filter := core.Filter{
+		PodName:             c.Query("podName"),
+		PodNamespace:        c.Query("podNamespace"),
+		StartTimeBeginStr:   c.Query("startTimeBegin"),
+		StartTimeEndStr:     c.Query("startTimeEnd"),
+		FinishTimeBeginStr:  c.Query("finishTimeBegin"),
+		FinishTimeEndStr:    c.Query("finishTimeEnd"),
```

Comment:
I think it is too complicated now... :) Do we need to have begin and end range for start and finish? TBH, the use case is not much.
Even in Grafana, it only uses start time and end time, it is not necessary to support range for both.

Reply:
Because the time saved in the database is accurate to the second, if the interval query is not provided and the user directly specifies a time, then the time may not be accurate to the second. If it is an interval, the user does not need to provide the precise time, and the experience may be better.

Reply:
@g1eny0ung Hello, would you mind sharing your idea about this? Since you are a direct user of this API.

Reply:
@yeya24 @fewdan First, I'm okay with this implementation too. But actually...I also think it's a little complicated for "users". The users represent developers who want to use chaos mesh API to develop something.

And for front-end usage, I think I will use the `StartTimeBeginStr` and `FinishTimeEndStr`, According to my understanding, at the time point of `StartTimeEndStr`, this event has already started. Same for `FinishTimeBeginStr`.

Reply:
@fewdan I think the precision is not very important in this API. So it is okay to just use just startTime and finishTime here. As @g1eny0ung also just needs the two time.

Adding the range time support for start and finish is weird and will cause confusion.